pkg/xds/sync/metrics.go

```diff
@@ -0,0 +1,35 @@
+package sync
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	core_metrics "github.com/kumahq/kuma/pkg/metrics"
+)
+
+type XDSSyncMetrics struct {
```

Comment:
Why not just `XDSMetrics` to be in the same fashion as `SDSMetrics`?

Reply:
refactored